control/veda_heal.py

```diff
@@ -85,7 +84,7 @@ def send_encodes(self):
                     jobid = uuid.uuid1().hex[0:10]
                     celeryapp.worker_task_fire.apply_async(
                         (veda_id, encode_profile, jobid),
-                        queue=self.auth_dict['celery_worker_queue'].split(',')[0]
+                        queue=self.auth_dict['celery_worker_queue']
```

Comment:
this always confuses me. `celery_worker_queue` is defined as comma separated string with 2 queue names in it. how the multiple queue string will work in the above case? why we don't want to use `.split(',')[0]`? Shouldn't we have only single queue name for `celery_worker_queue` variable?

Reply:
@muhammad-ammar We should only have one queue name. This was a holdover from a previous iteration where we had different queues for filesizes, but we don't need it any more (we just upped the EC2 instance storage.)
It's weird, because celery consumers and celery enqueuers read the string differently, and it's cause enqueueing problems in the past. 
So, one queue. That should make the machines be less confused.

Reply:
@muhammad-ammar https://github.com/edx-ops/veda-secure/pull/11

Reply:
@yro `celery_worker_queue` is unchanged in the `static_config.yaml` and static configs overwrite the `intstance_config.yaml` today. So, this will crash is production.

Reply:
@Qubad786  This is exactly the situation I was trying to avoid. We shouldn't have this be a static config. What if we want to have different priority queues in the future? This is a flaw, but I'll make the change to get this shipped.

Reply:
Right on -- I'll leave it for now. Thanks!